sources/presentation/Stride.Core.Presentation/Themes/ThemeSelector.xaml

```diff
@@ -70,59 +70,63 @@
-                  <Button Content="{me:Image {StaticResource VectorRestoreWindow}, 12, 12, NearestNeighbor}" Width="30" Height="25" Background="Transparent" x:Name="RestoreButton" DockPanel.Dock="Right" HorizontalAlignment="Center" VerticalAlignment="Center" Command="{x:Static cmd:SystemCommands.RestoreWindowCommand}" CommandParameter="{Binding RelativeSource={RelativeSource TemplatedParent}}" WindowChrome.IsHitTestVisibleInChrome="True"/>
-                  <Button Content="{me:Image {StaticResource VectorMinimizeWindow}, 12, 12, NearestNeighbor}"  Width="30" Height="25" Background="Transparent" x:Name="MinimizeButton" DockPanel.Dock="Right" HorizontalAlignment="Center" VerticalAlignment="Center" Command="{x:Static cmd:SystemCommands.MinimizeWindowCommand}" CommandParameter="{Binding RelativeSource={RelativeSource TemplatedParent}}" WindowChrome.IsHitTestVisibleInChrome="True"/>
-                  <Button DockPanel.Dock="Left" Margin="4,0" Padding="0" Background="Transparent" Width="28" Height="25" VerticalAlignment="Center" VerticalContentAlignment="Center" Command="{x:Static cmd:SystemCommands.ShowSystemMenuCommand}" CommandParameter="{Binding RelativeSource={RelativeSource TemplatedParent}}" WindowChrome.IsHitTestVisibleInChrome="True">
-                    <Button.Template>
-                      <ControlTemplate TargetType="Button">
-                        <ContentPresenter Content="{TemplateBinding Content}" HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}" Margin="{TemplateBinding Padding}" VerticalAlignment="{TemplateBinding VerticalContentAlignment}" RecognizesAccessKey="True"/>
-                      </ControlTemplate>
-                    </Button.Template>
-                    <Image Source="{DynamicResource VectorEditorIcon}" Width="22" Opacity="0.5"/>
-                  </Button>
-                  <TextBlock VerticalAlignment="Center" Text="{TemplateBinding Title}" Margin="0,4" FontSize="13"/>
-                </DockPanel>
-                <Border BorderBrush="{TemplateBinding BorderBrush}" BorderThickness="{TemplateBinding BorderThickness}" Background="{TemplateBinding Background}">
-                  <AdornerDecorator>
-                    <ContentPresenter Content="{TemplateBinding Content}" ContentTemplate="{TemplateBinding ContentTemplate}" />
-                  </AdornerDecorator>
-                </Border>
+          <Border x:Name="ContentBorder" Background="{TemplateBinding Background}" BorderBrush="{DynamicResource MainWindowActiveDefaultBorderBrush}" BorderThickness="1" IsHitTestVisible="True">
+            <DockPanel>
+              <DockPanel x:Name="TitleBar" DockPanel.Dock="Top" VerticalAlignment="Top" WindowChrome.IsHitTestVisibleInChrome="False" Height="32">
+                <Button Style="{DynamicResource MainWindowButtonStyle}" DockPanel.Dock="Right" Content="{me:Image {StaticResource VectorCloseWindow}, 12, 12}" x:Name="CloseButton" Command="{x:Static cmd:SystemCommands.CloseWindowCommand}"/>
```

Comment:
Any reasons for these to use dynamic resource binding? Dynamic resources are more expensive as they can't be cached and affect dramatically the performance. As much as possible, static resources should be used.

Reply:
I was going with static resources first, but then thought about the possibility of changing themes at runtime later on (as in VS). Right now you'd have to restart the editor.
I can change it to static resources so the change to dynamic resources can be made when it's required. I'll probably have to reorder some definitions then to appear in front of where they're used.

Reply:
Yes please assume live theme switching won't be supported anytime soon (seems too heavy because of dynamic resources) and switch back to StaticResource.
We can always change it back if we decide to allow live theme switching, but it seems highly unlikely so far.

Reply:
Okay, I should also switch to dynamic color resources in the AvalonDock theme then (which I'm updating right now in another branch to the most recent version) to make it consistent.

From what I know, colors are typically dynamic resources to also reflect system color changes (which became kinda rare, to be fair), but styles should indeed be static, unlike done above.

Reply:
Yes, restart for theme change is totally fine... Great to set this morning forward!

Reply:
> From what I know, colors are typically dynamic resources to also reflect system color changes (which became kinda rare, to be fair), but styles should indeed be static, unlike done above.

That's a good point. I'm wondering if we should also change them to be static. It probably needs some testing and performance comparison especially when working on a big project with lots of resources/assets and entities in a scene to see if it has any impact.

Let's keep the new resources in the theme static, and the color dynamic when they already were for now. We can review it later.